src/context/slider.tsx

```diff
@@ -16,8 +16,8 @@ const SliderContext = React.createContext<SliderContextType>({
   currentSlide: 3,
   translate: 0,
   movies: [],
-  nextImg: () => {},
-  prevImg: () => {},
+  nextImg: noop,
+  prevImg: noop,
```

Comment:
```suggestion
  nextImg: noop(),
  prevImg: noop(),
```

You have to call it so it will return the function.